src/messaging-component.js

```diff
@@ -0,0 +1,115 @@
+/* globals ppq */
+/* @flow */
+
+import { Tracker } from './tracker-component';
+import { checkIfMobile } from './lib/mobile-check';
+import { loadJavascript } from './lib/load-js';
+
+const museSdkUrl = 'https://ppsong.c0d3.com/cdn/assets/modal/sdk.js';
+loadJavascript(museSdkUrl);
+
+let userId = '';
+let isRendered = false;
+
+// $FlowFixMe
+ppq('showExperience', 'https://www.paypalobjects.com/muse/cart-recovery-0.3/', 'body', {
```

Comment:
We should avoid kicking off side-effects in the module scope. It makes testing tricky, and also de-optimizes any tree shaking that can be done by webpack.

The SDK allows you to export a `setup` function which runs on load. Can we use that instead? For example: https://github.com/paypal/paypal-checkout-components/blob/master/src/interface/button.js#L51